Rootlog_V2.md

```diff
@@ -614,9 +614,9 @@ The Keepers in Iron's Faction Board is its Relics.
 - `t_f` = Figure relic token
 - `t_t` = Tablet relic token
 - `t_j` = Jewelry relic token
-- `b_f` = Waystation, figure side up
```

Comment:
Could we keep these three in as well? I don't like the idea of requiring people to notate which waystation side is down. So, it would be, `b_f` means figure side up, unknown side down, and `b_f_<x>` means a specific waystation side is down, and we know what it is.

Reply:
Good spot. Yes, we can keep both notations.